src/App.js

```diff
@@ -3,18 +3,51 @@ import React from "react";
 
 const IMAGE_HEIGHT = 200;
 
-class Image extends React.Component {
+class ImageList extends React.Component {
+
+    render() {
+        const cats = this.props.cats;
+        return (
+            <div>
+                {cats}
```

Comment:
Similar to the previous case with the state, I think it's better to pass the data itself (i.e. the `images` array) into the props instead of passing the array of React elements. For example, passing the data is more flexible, because in the future you can use/read the data for some new purposes, while it's quite little you can do with the React elements.

Reply:
Another example why passing the data would be better: in the future you may add a new prop like `renderMode`, which will control how the image array will be rendered (e.g. in a simple list or in a grid).